tiup/tiup-reference.md

```diff
@@ -0,0 +1,87 @@
+---
+title: TiUP Reference
+---
+
+# TiUP Reference
+
+TiUP serves as the package manager of the TiDB ecosystem. It manages components in the TiDB ecosystem, such as TiDB, PD, and TiKV.
+
+## Syntax
+
+```shell
+tiup [flags] <command> [args...]        # Executes a command
+# or
+tiup [flags] <component> [args...]      # Runs a component
+```
+
+You can use the `help` command to get the information of a specific command. The summary of each command shows its parameters and their usage. Mandatory parameters are shown in angle brackets, and optional parameters are shown in square brackets.
+
+`<command>` represents the command name. For the list of supported commands, see the [Command list](#command-list) below. `<component>` represents the component name. For the list of supported components, see the [Component list](#component-list) below.
+
+## Options
+
+### -B, --binary
+
+- Data type: `BOOLEAN`
```

Comment:
Maybe change the order of the list to keep it consistent with the form of other prs.

Reply:
Done in https://github.com/pingcap/docs/pull/5322/commits/e2a2c1651c6f4ddfa9165e4c9d39913b02b45175.